includes/publisher/js/src/components/Field.jsx

```diff
@@ -28,6 +30,24 @@ export default function Field(props) {
 		let error = defaultError;
 
 		if (field.type === "number") {
+			if (!integerRegex.test(event.target.value)) {
+				(error = __(
+					"Value must be an integer.",
+					"atlas-content-modeler"
+				)),
+					event.target.max.toString();
```

Comment:
`event.target.max.toString()` seems left over from the original `sprintf` and can be removed, as in the original suggestion from https://github.com/wpengine/atlas-content-modeler/pull/458#discussion_r825979838:

```suggestion
				error = __(
					"Value must be an integer.",
					"atlas-content-modeler"
				);
```